lib/cc/analyzer/include_paths_builder.rb

```diff
@@ -0,0 +1,140 @@
+module CC
+  module Analyzer
+    class IncludePathsBuilder
+      def initialize(cc_excludes)
+        @cc_excludes = cc_excludes
```

Comment:
Maybe `cc_exclude_paths`? I wanted to differentiate it from patterns in `.gitignore`, which aren't ever technically called `exclude_paths`, but are usually a list of paths to exclude.